spec/domain/export/tabular/people/sac_recipient_household_row_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require "spec_helper"
+
+describe Export::Tabular::People::SacRecipientHouseholdRow do
+  let(:group) { groups(:bluemlisalp_mitglieder) }
+  let(:person) do
+    Fabricate.build(
+      :person,
+      id: 42,
+      first_name: "Hans",
+      last_name: "Muster",
+      street: "Musterstrasse",
+      housenumber: "42",
+      zip_code: "4242",
+      town: "Musterhausen",
+      country: "Schweiz",
+      email: "hans.muster@example.com"
+    )
+  end
+  let(:family_member) do
+    Fabricate.build(:person,
+      id: 43,
+      first_name: "Max",
+      last_name: "Muster",
+      email: "max.muster@example.com",
+      street: "Maxweg",
+      housenumber: "12",
+      household_key: "42")
+  end
+
+  subject(:row) { described_class.new(people, group) }
+
+  def value(key) = row.fetch(key)
+
+  describe "person without household" do
+    let(:people) do
+      [person]
+    end
+
+    it("id") { expect(value(:id)).to eq 42 }
+    it("salutation") { expect(value(:salutation)).to be_nil }
+    it("first_name") { expect(value(:first_name)).to eq "Hans" }
+    it("last_name") { expect(value(:last_name)).to eq "Muster" }
+    it("adresszusatz") { expect(value(:adresszusatz)).to be_nil }
+    it("address") { expect(value(:address)).to eq "Musterstrasse 42" }
+    it("postfach") { expect(value(:postfach)).to be_nil }
+    it("zip_code") { expect(value(:zip_code)).to eq "4242" }
+    it("town") { expect(value(:town)).to eq "Musterhausen" }
+    it("email") { expect(value(:email)).to eq "hans.muster@example.com" }
+    it("layer_navision_id") do
+      expect(value(:layer_navision_id)).to eq groups(:bluemlisalp).navision_id
```

Comment:
ich fände das jetzt einiges leserlicher wenn man einfach die row als hash vergleicht als das man hier `it` so viel verwendet. hier hat man so viele expect und value aufrufe